app/models/jurisdiction.rb

```diff
@@ -20,4 +20,8 @@ def display_full
     result.concat(" (#{abbr})") unless abbr.blank?
     result
   end
+
+  def self.pluck_h(keys)
```

Comment:
Because instead converting **to a hash** with **to_h** I want to **pluck a hash** with pluck_h.

Plus it was the example I found on-line! ;)

But I am going to look at re-writing this whole block anyway...